koocook_core/tests/test_support/test_markdown.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+
+from ...support.markdown import MarkdownSource
+
+
+# TODO: Preliminary tests
+class TestMarkdown(unittest.TestCase):
+    def test_proper_escape(self):
+        source = MarkdownSource("\"**Test**\"")
+        self.assertEqual(source.rendered, "<p>&quot;<strong>Test</strong>&quot;</p>")
+
+    def test_escaped(self):
+        source = MarkdownSource("&lt;&gt;")
+        self.assertEqual(source.rendered, "<p>&lt;&gt;</p>")
```

Comment:
More tests would be welcomed. Is this PR a requirement for others?

Reply:
PR Prerequisite to: (in sequence)

- `feature/rating`
- `feature/popularity`
- `feature/personalisation`

Reply:
😢

Reply:
I've added unit tests! Feel free to check them out.